addons/docs/src/mdx/__snapshots__/mdx-compiler-plugin.test.js.snap

```diff
@@ -51,6 +51,8 @@ function MDXContent({ components, ...props }) {
 
 MDXContent.isMDXComponent = true;
 
+const makeStoryFn = val => (typeof val === 'function' ? val : () => val);
```

Comment:
Maybe this should be an import? Or defined inline?

Reply:
fixed